test/integration/test_bind.rb

```diff
@@ -10,7 +10,9 @@ def test_bind_timeout
     error = assert_raise Net::LDAP::Error do
       @ldap.bind(method: :simple, username: "uid=user1,ou=People,dc=rubyldap,dc=com", password: "passworD1")
     end
-    assert_equal('Connection timed out - user specified timeout', error.message)
+    msgs = ['Operation timed out - user specified timeout',
+            'Connection timed out - user specified timeout']
+    assert_send([msgs, :include?, error.message])
```

Comment:
> Would it make more sense for #bind and other bits to normalize the timeout error message, so it's the same across all platforms?

:+1: I like this. It'd be less surprising as a user of this library. Up to you whether you want to do it in this PR, or follow up with a separate one.